website/docs.md

```diff
@@ -2624,6 +2624,15 @@ We believe that the Service Weaver's *modular monolith* model has the best of
 both worlds: the ease of development of monolithic applications, with the
 runtime benefits of microservices.
 
+### Service Weaver on macOS
```

Comment:
Sorry, don't know where my earlier comment went. Can we put this as a `Note` somewhere in the Installation section?

Reply:
Good idea, done.